pom.xml

```diff
@@ -73,15 +74,25 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
```

Comment:
I think you can remove vaadin-bom since hilla already has it

Reply:
Sounds like an internal (problem) of Hilla that will be resolved at some point?